tests/test_c_generator.py

```diff
@@ -339,10 +339,14 @@ def test_pragma(self):
             void f() {
                 #pragma bar
                 i = (a, b, c);
+                if (d)
```

Comment:
Can you please add separate tests here with 1 pragma in the block, and two pragmas in the block.

Reply:
Done.
Rebase comment should also be ok.